File: WWW/DesktopModules/NVCMS.FAQs/Manager/FAQs/Question/Main.ascx.vb
Imports System
Imports System.Web
Imports System.Web.UI
Imports System.Web.UI.WebControls
Imports DotNetNuke
Namespace NVCMS.Modules.FAQs
    Partial Class ManagerMain
        Inherits Entities.Modules.PortalModuleBase

        Protected Sub Page_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles Me.Load
            Try
                'If Not IsPostBack Then

                Dim DynamicPage As String
                Select Case Request.Item("view")
                    Case "add", "edit"
                        DynamicPage = DotNetNuke.Common.ResolveUrl(Me.TemplateSourceDirectory & "/_edit.ascx")
                    Case Else
                        DynamicPage = DotNetNuke.Common.ResolveUrl(Me.TemplateSourceDirectory & "/_view.ascx")
                End Select
                Dim objModule As Entities.Modules.PortalModuleBase = CType(Me.LoadControl(DynamicPage), DotNetNuke.Entities.Modules.PortalModuleBase)
                If Not objModule Is Nothing Then
                    objModule.ModuleConfiguration = Me.ModuleConfiguration
                    phDynamicPlaceHolder.Controls.Add(objModule)
                End If
                'End If
            Catch ex As Exception
                ProcessModuleLoadException(Me, ex)
            End Try
        End Sub

    End Class
End Namespace

File: WWW/DesktopModules/NVCMS.LadingPage/Display/Main.ascx.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;
using System.IO;
using System.Text.RegularExpressions;
using DotNetNuke;
using DotNetNuke.Entities;
using DotNetNuke.Entities.Modules;
using DotNetNuke.Common;
using DotNetNuke.Entities.Portals;
using DotNetNuke.Common.Utilities;
using NVCMS.Modules.TinTuc;
namespace NVCMS.Modules.LadingPage
{
    public partial class Main : PortalModuleBase
    {
        protected void Page_Load(object sender, EventArgs e)
        {
            string fbclid = Request.QueryString["fbclid"];
            string sUrl = Request.RawUrl.Replace("?fbclid=" + fbclid, "");
			string gidzl = Request.QueryString["gidzl"];
            sUrl = Request.RawUrl.Replace("?gidzl=" + gidzl, "");
			
            int sId = Ultis.GetRequestId(sUrl);
            PortalModuleBase control = new PortalModuleBase();
            if (sId > 0) //Index
            {
                control = Page.LoadControl("~/DesktopModules/NVCMS.LadingPage/Display/Detail.ascx") as PortalModuleBase;
               
            }
            else  //Details
            {
                control = Page.LoadControl("/DesktopModules/NVCMS.LadingPage/Display/Index.ascx") as PortalModuleBase;
            }
            //Response.Write(sId);
            control.ModuleConfiguration = ModuleConfiguration;
            plhNews.Controls.Add(control);
        }
    }
}